src/diamond/handler/pubsub.py

```diff
@@ -0,0 +1,216 @@
+# coding=utf-8
+
+"""
+Send metrics to [Google Pub/Sub](https://cloud.google.com/pubsub/).
+
+#### Copyright
+
+Copyright 2016 Google Inc.
```

Comment:
I talked to legal here to make sure I'm speaking correctly.  I can remove the copyright verbiage from the pubsub.py file but it needs to stay in the main LICENSE.  A direct quote from legal is:  "This is not because Google endorses the project per se, but because we're a copyright author.  Brightcove Inc. is already listed in the LICENSE, the the project has demonstrated an understanding of corporate copyright authorship you can point to."

Hope that clears it up.